### DISCUSSION TITLE
https://blog.3place.cn/read/%E5%95%86%E5%90%9B%E4%B9%A6-%E7%BE%A4%E5%86%85%E4%BA%A4%E6%B5%81.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#2] [General]
# https://blog.3place.cn/read/%E5%95%86%E5%90%9B%E4%B9%A6-%E7%BE%A4%E5%86%85%E4%BA%A4%E6%B5%81.html

https://blog.3place.cn/read/%E5%95%86%E5%90%9B%E4%B9%A6-%E7%BE%A4%E5%86%85%E4%BA%A4%E6%B5%81.html

**Comment by hbaolong** [comment]
我要留个言呀。哈哈。

**Comment by picklecai** [comment]
商鞅为什么不直接回答杜挚的问题呢？

**Reply by hbaolong** [reply]
因为，第一个问题，不值得回答。

**Comment by hbaolong** [comment]
再来一个留言

**Comment by picklecai** [comment]
坐看小世界成长为大世界
